includes/classes/ShowData.php

```diff
@@ -0,0 +1,55 @@
+<?php 
+class ShowData extends base
```

Comment:
Can we have a better class name here, that is more descriptive of the action we are doing